app/cu_editor.py

```diff
@@ -396,11 +396,49 @@ def setTextBuffer(self, textBuffer):
       KEY_F2: textBuffer.bookmarkNext,
       KEY_F3: textBuffer.findAgain,
       KEY_F4: self.changeToPaletteWindow,
+      KEY_F5: self.changeToPopup,
```

Comment:
A test key is okay, but consider commenting it out for check-in and/or make a note that this is for testing.

Reply:
Oops forgot to remove this. I'll edit it in the next PR. Sorry about that.